argmax/Cargo.toml

```diff
@@ -16,5 +16,5 @@ wit-bindgen-rust = { git = "https://github.com/bytecodealliance/wit-bindgen", op
 crate-type = ["rlib", "cdylib"]
 
 [features]
-default = []
+default = ["metadata"]
```

Comment:
Please revert this change. Enabling the `metadata` feature by default will break any Runes that use this proc-block because of the environment a Rune is compiled in (`no_std`, etc.).